test/index.js

```diff
@@ -311,6 +328,9 @@ test('headers :: custom', t => {
 					{ type: 'script', href: '/bundle.1234.js' },
 					{ type: 'style', href: '/bundle.612d.css' },
 					{ type: 'image', href: '/link.svg' },
+					{ type: 'script', href: '/bundle.5678.js' },
+					{ type: 'style', href: '/bundle.890d.css' },
+					{ type: 'image', href: '/button.svg' },
```

Comment:
What are all these extra files? Why are they showing up alongside the previous `bundle.*` (shared) files?

Reply:
@lukeed 
The newly added file is a test case for webpack 5. This is the case when the entry contains a string (second argument)

https://github.com/lukeed/webpack-route-manifest/blob/bb0774f75df328695646409b41c661b0e2577a1b/test/index.js#L27-L28

I think these files need to be added to ensure the correctness of this implementation.

Reply:
Ideally this would be separated into a `webpack5` namespace so that the webpack4 results weren't mixed in with the webpack5 results, because it's confusing & they wouldn't be living alongside each other in a real app.

That said, I don't blame you for not doing it because it'd be really tedious and annoying

Reply:
> Ideally this would be separated into a webpack5 namespace

I see.
Do you mean to get the version of webpack that the application is currently using from the stats information and create it based on that?

Reply:
No – these tests has a mocking system in place that replicates the Webpack 4 module graph layout. Your changes introduced a Webpack 5 module graph, and the tests are dealing with e `DEFAULTS` graph that has Webpack 4 _and_ Webpack 5 modules inside of it. This will never happen in an actual build, and that's why the output looked confusing/verbose.

Reply:
Indeed, this test is confusing.
This test case looks strange because the application will never use both webpack4 and webpack5.
Thanks for the review!